src/test/java/edu/cornell/kfs/vnd/batch/service/impl/CuCommodityCodeUpdateServiceImplTest.java

```diff
@@ -1,45 +1,56 @@
-import org.kuali.kfs.sys.context.KualiTestBase;
-import org.kuali.kfs.sys.context.SpringContext;
+import java.util.ArrayList;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
 import org.kuali.kfs.vnd.businessobject.CommodityCode;
-import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
 
-import edu.cornell.kfs.vnd.batch.service.CommodityCodeUpdateService;
+import edu.cornell.kfs.sys.batch.service.impl.CuBatchInputFileServiceImpl;
+import edu.cornell.kfs.vnd.batch.CommodityCodeInputFileType;
 
-@ConfigureContext
-public class CuCommodityCodeUpdateServiceImplTest extends KualiTestBase {
+public class CuCommodityCodeUpdateServiceImplTest {
 	
-	private CommodityCodeUpdateService commodityCodeUpdateService;
-    private ConfigurationService  kualiConfigurationService;
+	private CommodityCodeUpdateServiceImpl commodityCodeUpdateService;
 
 	private static final String DATA_FILE_PATH = "src/test/resources/edu/cornell/kfs/vnd/fixture/commodityCodeFlatFile.txt";
+	private static final int NUMBER_OF_RECORDS_IN_DATA_FILE = 53321;
     
-	@Override
-	protected void setUp() throws Exception {
-		super.setUp();
-		commodityCodeUpdateService = SpringContext.getBean(CommodityCodeUpdateService.class);
+	@Before
+	public void setUp() throws Exception {
+		commodityCodeUpdateService = PowerMockito.spy(new CommodityCodeUpdateServiceImpl());
+		PowerMockito.doReturn(true).when(commodityCodeUpdateService, "updateCommodityCodes", Mockito.any());
+		
+		CuBatchInputFileServiceImpl batchInputFileService = new CuBatchInputFileServiceImpl();
+		commodityCodeUpdateService.setBatchInputFileService(batchInputFileService);
+		
+		CommodityCodeInputFileType commodityCodeInputFileType = new CommodityCodeInputFileType();
+		commodityCodeUpdateService.setCommodityCodeInputFileType(commodityCodeInputFileType);
 	}
 	
+	@After
+	public void tearDown() {
+	    commodityCodeUpdateService = null;
+	}
+	
+	@Test
 	public void testLoadCommodityCodeFile() {
-		byte[] zeroLength = {};
-		ArrayList<CommodityCode> noList = new ArrayList();
-		byte[] noContent = commodityCodeUpdateService.getFileContent("no such file");
-		ArrayList<CommodityCode> noCommodityList = (ArrayList<CommodityCode>) commodityCodeUpdateService.parseCommodityCodeList(zeroLength);
 		byte[] content = commodityCodeUpdateService.getFileContent(DATA_FILE_PATH);
 		ArrayList<CommodityCode> someCommodityCodes = (ArrayList<CommodityCode>) commodityCodeUpdateService.parseCommodityCodeList(content);
 		
-		assertEquals(noContent.length, zeroLength.length);
-		assertEquals(noCommodityList.size(), noList.size());
 		assertNotSame(content.length, 0);
-		assertNotSame(someCommodityCodes.size(), 0);
-		
-		assert(commodityCodeUpdateService.loadCommodityCodeFile(DATA_FILE_PATH));
+		assertEquals(NUMBER_OF_RECORDS_IN_DATA_FILE, someCommodityCodes.size());
+	}
+	
+	@Test
+	public void testloadCommodityCodeFile() {
```

Comment:
This method has almost the same name as another test method in this class, where the only difference is that the first "L" has been lowercased. I would suggest renaming this method and/or the other method accordingly to avoid confusion.

Reply:
Thanks, updated function names